sigs.yaml

```diff
@@ -3467,6 +3467,47 @@ workinggroups:
     liaison:
       github: pohly
       name: Patrick Ohly
+- dir: wg-serving
+  name: Serving
+  mission_statement: >
+    Discuss and enhance the support of inference serving for accelerated workloads
+    in Kubernetes. Make Kubernetes the natural choice for hosting production inference
+    reliably, and improve all serving workloads along the way.
+
+  charter_link: charter.md
+  stakeholder_sigs:
+  - Apps
+  - Architecture
+  - Autoscaling
+  - Instrumentation
+  - Network
+  - Node
+  - Scheduling
+  - Storage
+  label: serving
+  leadership:
+    chairs:
+    - github: SergeyKanzhelev
+      name: Sergey Kanzhelev
+      company: Google
+    - github: TBD
+      name: TBD
+      company: TBD
+  meetings:
+  - description: Regular Meeting ([Calendar](https://calendar.google.com/calendar/embed?src=TODO))
+    day: TODO
+    time: TODO
+    tz: PT (Pacific Time)
+    frequency: monthly
+    url: https://zoom.us/j/TODO
+    archive_url: https://docs.google.com/document/d/1aExJFtaLnO-TM6_2uILgI8NI0IjOm7FcwLABBKEMEo0/edit
+    recordings_url: https://www.youtube.com/playlist?list=TODO
+  contact:
+    slack: wg-serving
+    mailing_list: https://groups.google.com/a/kubernetes.io/g/wg-serving
+    liaison:
```

Comment:
@kubernetes/steering-committee what is the right way to get a liason for the WG?

Reply:
steering will discuss and assign once the charter is approved and merged

Reply:
so keep it as TODO for now?

Reply:
Yeah just leave it TODO and we'll discuss in steering and update these after merge

We should probably do a round of re-balancing liasons as well.